src/module_EARTH_GRID_COMP.F90

```diff
@@ -248,14 +248,15 @@ SUBROUTINE EARTH_REGISTER(EARTH_GRID_COMP,RC_REG)
         rc=RC)
       ESMF_ERR_RETURN(RC,RC_REG)
 
+#ifdef NEMS_DRIVER
```

Comment:
Mark, to avoid the code changes in ufs-weather-model side, can you change to:

ifndef JEDI_DRIVER

I assume JEDI would be the driver if the model runs within JEDI, by default, NEMS will be the driver.